### ISSUE TITLE
Collaborating / merging with gatsby-theme-ghost-casper

### ISSUE THREAD

**Issue by Vacilando** [scttcper/gatsby-theme-casper#3]
Where does this Gatsby theme stand in comparison with https://github.com/sasivarnan/gatsby-theme-ghost-casper? Any chance of merging and collaborating?

**Comment by styxlab** [comment]
As Ghost started a new [gatsby casper theme](TryGhost /
gatsby-starter-ghost) themselves, I also wonder how this project relates to these new developments.

**Comment by scttcper** [comment]
gatsby-starter-ghost from ghost reads from ghost's api to create gatsby static pages is not casper.

https://github.com/TryGhost/gatsby-starter-ghost seems to be pretty much the same thing, but using the styles from casper directly.

**Comment by styxlab** [comment]
Yes, the import from ghost CMS is out of your scope, but the theme part is definitely overlapping. Your theme actually looks closer to the original Casper theme than theirs, so, maybe you can join forces? Would love to see a fully production ready Gatsby Casper v3 theme emerging soon.
